common.go

```diff
@@ -23,6 +23,7 @@ var DefaultStrategies = []Strategy{
 	GetLanguagesByShebang,
 	GetLanguagesByExtension,
 	GetLanguagesByManpage,
+	GetLanguagesByXML,
```

Comment:
It would be nice to match the order of strategies that Linguist is [using](https://github.com/github/linguist/blob/7c2adbdb15d4efd25d92cd5ed20d0025f0d32d28/lib/linguist.rb#L67)
<img width="506" alt="Screenshot 2020-11-26 at 12 43 04" src="https://user-images.githubusercontent.com/5582506/100346963-f090c300-2fe4-11eb-98f5-267f67ed5ca3.png">

Reply:
Fixed